src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="NoCompatibleTargetFramework" xml:space="preserve">
-    <value>Cannot find a compatible framework for '{0}' in the following frameworks: {1}.</value>
+    <value>Project '{0}' targets '{2}'. It cannot be referenced by a project that targets '{1}'.</value>
```

Comment:
Looks like the placeholder ordering is different here than the call site.

Reply:
I reordered the callsite, so the message is:
`Project '{0}' targets '{2}'. It cannot be referenced by a project that targets '{1}'.`
and the callsite is:
`string.Format(Strings.NoCompatibleTargetFramework, project.ItemSpec, CurrentProjectTargetFramework, targetFrameworks)`

It looked correct in the unit tests.

Reply:
You're right, I read wrong.